src/webviews/util.ts

```diff
@@ -3,6 +3,13 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+/**
+ * IMPORTANT:
+ *
+ * This class is used in the frontend vue.js webview code.
+ * Do not put backend code in this module.
+ */
```

Comment:
added an item in https://github.com/aws/aws-toolkit-vscode/issues/2347 to track adding a lint rule for this.